R/train_models.R

```diff
@@ -244,7 +244,7 @@ train_models <- function(run_info,
     x = current_combo_list_final,
     .combine = "rbind",
     .packages = packages,
-    .errorhandling = "stop",
+    .errorhandling = "remove",
```

Comment:
Please change this back to stop. We want to make sure all time series run, and if they don't then return an error